src/components/ChatLog.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+
+
+const ChatLog = ({entries, updateChatData}) => {
```

Comment:
Same as before, make sure you are bringing these in as props and not as a defined set of parameters!